client/my-sites/stats/stats-navigation/index.jsx

```diff
@@ -34,7 +34,7 @@ const StatsNavigation = ( props ) => {
 			statsControl = (
 				<SegmentedControl
 					// eslint-disable-next-line wpcalypso/jsx-classname-namespace
-					className="stats-navigation__control"
+					className="stats-navigation__control for-woocommerce"
```

Comment:
Having said that, I do like what you've done here.

![screen shot 2017-06-14 at 12 14 09 pm](https://user-images.githubusercontent.com/1922453/27110105-0d474a86-50fb-11e7-9ac2-42d92766a936.png)